docker-compose.yml

```diff
@@ -0,0 +1,154 @@
+version: "2.3"
```

Comment:
Do we have a particular reason to use version 2 syntax? Why not version 3?

Reply:
I'm more familiar with 2. During development there where some things that are removed in 3 that I used to use.

I will see if I can upgrade to 3.